packages/mobile/src/screens/feed-screen/FeedScreen.tsx

```diff
@@ -27,7 +27,10 @@ const messages = {
 
 export const FeedScreen = () => {
   const dispatchWeb = useDispatchWeb()
-  const feedLineup = useSelectorWeb(getFeedLineup, isEqual)
+  const feedLineup = useSelectorWeb(getFeedLineup, (a, b) => {
+    const omitUneeded = o => omit(o, ['inView'])
```

Comment:
nice find here, i wonder if we want this in other places too, can def check profile